### ISSUE TITLE
Non-TV Case

### ISSUE THREAD

**Issue by Marco-Christiani** [mlysy/kalmantv#6]
Is it possible to run a non-tv filter for a linear system? I have a script running a filter with great accuracy that is attempting to model a constant variance scenario, however the state means are not similar to results from other filters (filterpy/pykalman) and exhibit odd behaviors (assuming this is modeling nonlinear relationship). Seems as though I cannot seem to properly adapt the settings for the constant case, which embarrassingly sounds trivial. Is there a simple solution or any examples I could be pointed at?

Edit: Got it working. This implementation is blazingly fast and very accurate. Great work

**Comment by mlysy** [comment]
So glad this is useful for you!  We'll be upgrading to a [**JAX**](https://jax.readthedocs.io/) version of the library at some point.  It should be just as fast but amenable to automatic differentiation :)
